sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -305,6 +305,30 @@ def apply_minp(arr, ddof, minp):
     gen_hpat_pandas_series_rolling_ddof_impl(arr_var))
 
 
+@sdc_register_jitable
+def pop_count(value, nfinite, result):
+    """Calculate the window sum without old value."""
+    if not numpy.isnan(value):
+        result -= 1
+
+    return nfinite, result
```

Comment:
I would implement it something like this:
```python
if numpy.isnan(value):
    return nfinite, result

return nfinite, result - 1
```

The same is applicable for `put_count`.